contracts/colony/ColonyStorage.sol

```diff
@@ -243,6 +243,7 @@ contract ColonyStorage is ColonyDataTypes, ColonyNetworkDataTypes, DSMath, Commo
 
   modifier onlyOwnExtension() {
     require(isOwnExtension(msgSender()), "colony-must-be-own-extension");
+    assert(msgSender() == msg.sender);
```

Comment:
I would put this first but that's just me.

Reply:
If it's first, it's hittable, and therefore not appropriate for an assert.